pages/common/surfraw.md

```diff
@@ -0,0 +1,25 @@
+# surfraw
+
+> Surfraw is a CLI to query a variety of web search engines.
+> It consists of a collection of elvi, each of which knows how to search a specific website.
+> More information: <http://surfraw.org/>.
+
+- Show the list of elvi:
+
+`surfraw -elvi`
+
+- Search using an elvi and open the results page in the browser:
+
+`surfraw {{elvi_name}} {{search_terms}}`
+
+- Show the elvi description and specific options using the alias:
+
+`sr {{elvi_name}} -local-help`
```

Comment:
Please use the long command in the guide to match the page name. Talk about the alias in an extra final example.
```suggestion
`surfraw {{elvi}} -local-help`
```